includes/blocks/course-theme/class-course-theme.php

```diff
@@ -33,7 +34,6 @@ public function __construct() {
 	 * @access private
 	 */
 	public function enqueue_block_assets() {
-		Sensei()->assets->enqueue( 'sensei-course-theme', 'css/sensei-course-theme.css' );
```

Comment:
It was removed here because this class is already being enqueued as part of the `Sensei_Course_Theme` class.